diff --git a/models/pose2mesh/coarsening.py b/models/pose2mesh/coarsening.py
index 2d9b690..8ca7a3e 100644
--- a/models/pose2mesh/coarsening.py
+++ b/models/pose2mesh/coarsening.py
@@ -155,7 +155,7 @@ def HEM_one_level(rr, cc, vv, rid, weights):
     nnz = rr.shape[0]
     N = rr[nnz - 1] + 1
 
-    marked = np.zeros(N, np.bool)
+    marked = np.zeros(N, bool)
     rowstart = np.zeros(N, np.int32)
     rowlength = np.zeros(N, np.int32)
     cluster_id = np.zeros(N, np.int32)
